File: 05_PassingCars.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # write your code in Python 3.6
    pass
    total=0
    b=[]

    for i in range(len(A)):
	    if (A[i]==0):
		    b.append(i);

    for j in range(len(b)):
	    total+=(len(A)-b[j]-1)-(len(b)-j-1);

    if (total<0 or total>1000000000):
	    return -1
    else:
	    return total

File: 05_PassingCars.cpp
int solution(vector<int> &A)
{
	int total=0;
	vector<int> b;

	for (size_t i=0; i<A.size(); i++)
		if (A[i]==0) b.push_back(i);

	for (size_t j=0; j<b.size(); j++)
		total+=(A.size()-b[j]-1)-(b.size()-j-1);

	if (total<0 || total>1000000000)
		return -1;
	else
		return total;
}

